src/devices/tuya.ts

```diff
@@ -1467,6 +1467,25 @@ const definitions: Definition[] = [
             await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_gbagoilo']),
+        model: 'TS0601-1-gang-switchboard',
+        vendor: 'TuYa',
+        description: '1 gang switch',
+        exposes: [e.switch().withEndpoint('l1').setAccess('state', ea.STATE_SET)],
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        endpoint: (device) => {
+            return {'l1': 1};
+        },
+        meta: {
+            multiEndpoint: true,
```

Comment:
Please remove multiEndpoint here, remove `endpoint: (device) => {` function` and change `state_l1` to `state`

Reply:
This device is a multi endpoint device. The other endpoints are not active

Reply:
That doesn't matter, only 1 endpoint is used from z2m perspective so this can be removed.